google/oauth2/gdch_credentials.py

```diff
@@ -97,23 +98,27 @@ def __init__(
         super(ServiceAccountCredentials, self).__init__()
         self._signer = signer
         self._service_identity_name = service_identity_name
+        self._project = project
         self._audience = audience
         self._token_uri = token_uri
         self._ca_cert_path = ca_cert_path
 
     def _create_jwt(self):
         now = _helpers.utcnow()
         expiry = now + JWT_LIFETIME
+        iss_sub_value = "system:serviceaccount:{}:{}".format(
+            self._project, self._service_identity_name
+        )
 
         payload = {
-            "iss": self._service_identity_name,
-            "sub": self._service_identity_name,
+            "iss": iss_sub_value,
```

Comment:
why changing the issuer and subject?

Reply:
GDCH token server requires this new format.